genesyscloud/tfexporter/genesyscloud_resource_exporter.go

```diff
@@ -1167,13 +1167,13 @@ func (g *GenesysCloudResourceExporter) sanitizeConfigMap(
 		case map[string]interface{}:
 			// Maps are sanitized in-place
 			currMap := val.(map[string]interface{})
-			_, res := g.sanitizeConfigMap(resourceType, "", val.(map[string]interface{}), currAttr, exporters, exportingState, exportingAsHCL, false)
+			_, res := g.sanitizeConfigMap(resourceType, resourceName, val.(map[string]interface{}), currAttr, exporters, exportingState, exportingAsHCL, false)
```

Comment:
I'm not sure why we weren't passing in the `resourceName` to the function before, but its required in order to make `resourceName` available to nested objects like the `reply_email_address` block.